usage-insights-launch.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Introducing usage monitoring
+description: 'Learn about the new Usage feature available in Xata'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/usage-insights-launch.jpg
+  alt: Xata
+author: Siobhán Meier, Ryan Slade
+date: 03-20-2022
+tags: ['launch-week', 'metrics', 'announcement']
+published: false
+slug: usage-metrics-launch
```

Comment:
Note: updated the slug to avoid reinforcing the word "insights" given prior comments.